daprdocs/content/en/dotnet-sdk-docs/dotnet-development/dotnet-development-tye.md

```diff
@@ -0,0 +1,67 @@
+---
+type: docs
+title: "Dapr .NET SDK Development with Project Tye"
+linkTitle: "Development"
+weight: 40000
+description: Learn about local development with Project Tye
+---
+
+## Project Tye
+
+[.NET Project Tye](https://github.com/dotnet/tye/) is a microservices development tool designed to make running many .NET services easy. Tye enables you to store a configuration of multiple .NET services, processes, and container images as a runnable application. 
+
+Tye is advantageous for a .NET Dapr developer because:
+
+- Tye has the ability to automate the dapr CLI built-in
+- Tye understands .NET's conventions and requires almost no configuration for .NET services
+- Tye can manage the lifetime of your dependencies in containers
+
+**Pro:** Tye can automate all of the steps described above. You no longer need to think about concepts like ports or app-ids.
+
+**Pro:** Since Tye can also manage containers for you, you can make those part of the application definition and stop the long-running containers on your machine.
```

Comment:
```suggestion
Pros/cons:
- **Pro:** Tye can automate all of the steps described above. You no longer need to think about concepts like ports or app-ids.
- **Pro:** Since Tye can also manage containers for you, you can make those part of the application definition and stop the long-running containers on your machine.
```